command/server/server.go

```diff
@@ -174,6 +182,45 @@ func (s *Server) setupMetrics(config *TelemetryConfig) error {
 
 	}
 
+	// setup open collector tracer
+	ctx := context.Background()
+
+	res, err := resource.New(ctx,
+		resource.WithAttributes(
+			// the service name used to display traces in backends
+			semconv.ServiceNameKey.String("bor"), // TODO: use the service name from the config
+		),
+	)
+	if err != nil {
+		log.Error("Failed to create open telemetry resource", "service", "bor")
+	}
+
+	// Set up a trace exporter
+	traceExporter, err := otlptracegrpc.New(ctx,
+		otlptracegrpc.WithInsecure(),
+		otlptracegrpc.WithEndpoint("localhost:"+strconv.FormatUint(config.OpenCollectorPort, 10)),
```

Comment:
I think we'll need to setup the OpenCollector address too, this could be just one param with the addr+port, as the collector can be also installed in a remote host.

Reply:
Yes, makes sense. So, should I change it to OpenTelemetry URL and use that instead of just port?

Reply:
I would call it the OpenCollectorEndpoint.

Reply:
Why not maintain consistency with the existing vars? `PrometheusAddr` https://github.com/maticnetwork/bor/pull/235/files#diff-0b1642f753dfe53bd60b0d3f731e4e7728f66e550b8820013a46e48db9c317b5R280

Reply:
But it's consistent with `WithEndpoint` so LGTM